src/devtools-window-polyfill.ts

```diff
@@ -5,6 +5,7 @@ import ws from 'ws';
 const customGlobal = global as any;
 
 // These things must exist before importing `react-devtools-core`
+// eslint-disable-next-line n/no-unsupported-features/node-builtins
```

Comment:
Unclear why this lint error just started to appear, but should be safe to ignore.